kolibri/plugins/learn/assets/src/vue/expandable-content-grid/index.vue

```diff
@@ -77,6 +77,10 @@
 
   .button-wrapper
     text-align: center
+    position: absolute
+    left: 50%
+    transform: translateX(-50%)
+    padding-bottom: 60px
```

Comment:
This causes the button to pop out of its parent's box. It works right now but will definitely break in the future, e.g. if there were any content under the `<expandable-content-grid>`.

The real problems with the button not aligning properly are:

1. The padding-left of `.main` in the main vue
2. `<expandable-content-grid>` has two root elements. It should instead have a single wrapper div. (this issue existed before your PR)